### DISCUSSION TITLE
路由前缀

### DISCUSSION THREAD

**Discussion by Immanding** [umijs/umi#11182] [Q&A]
<img width="272" alt="image" src="https://github.com/umijs/umi/assets/19871501/83ef3a9e-8c06-4475-bd95-7f703152313c">
![image](https://github.com/umijs/umi/assets/19871501/918fb20b-6f2e-4561-ad54-30984d1adc2d)
为什么路由会从package.json 的那么属性获取前缀，如何解决

**Comment by Immanding** [comment]
name 属性

**Comment by Immanding** [comment]
用了微前端插件

**Comment by fz6m** [comment]
微前端子应用单独开发的时候需要区分，会增加一个 name 的 base ，如果套在父应用里是没有这个 base 的。
